client/public/index.html

```diff
@@ -38,6 +38,8 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
+    <script type="module" src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.esm.js"></script>
+    <script nomodule src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.js"></script>
```

Comment:
it's not good to use cdn links for icons and styles with react
you can just install it in your app with `npm i ....`
and Marital UI has a great collection of icons
[MUI Icons](https://mui.com/material-ui/material-icons/)